MeshLib/Mesh.h

```diff
@@ -172,6 +173,23 @@ void scaleMeshPropertyVector(MeshLib::Mesh& mesh,
                              std::string const& property_name,
                              double factor);
 
+template <typename T>
+void addPropertyToMesh(MeshLib::Mesh& mesh, std::string const& name,
```

Comment:
Needs a docu especially about existing properties.

reserve and back_inserter below ;)